.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an enhancement or request support for a new language
```

Comment:
IMHO, this should be split into two templates:

* `feature_request.md`
* `new_language.md`

__Rationale:__ Not every user will equate _"new feature"_ with _"new language"_. Some may consider the latter to be extending an existing feature (highlighting, classification, etc). Languages make up the bulk of enhancement requests, anyway.

Reply:
Good idea. 👍